.github/workflows/OntologyTypeValidator.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow validates the ontology types
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Ontology Type Validator
+
+on:
+  push:
+    branches: [ master ]
+    paths: 
+      - 'ontology/yaml/resources/**'
+  pull_request:
+    branches: [ master ]
+    paths: 
+      - 'ontology/yaml/resources/**'
+
+jobs:
+  build:
+  
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest]
```

Comment:
no windows?

Reply:
no yet, @Ray725 is working the bug for windows